flow-decomposition/src/main/java/com/powsybl/flow_decomposition/rescaler/DecomposedFlowRescaler.java

```diff
@@ -8,12 +8,13 @@
 package com.powsybl.flow_decomposition.rescaler;
 
 import com.powsybl.flow_decomposition.DecomposedFlow;
+import com.powsybl.iidm.network.Network;
 
 /**
  * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
  * @author Hugo Schindler {@literal <hugo.schindler at rte-france.com>}
  * @author Caio Luke {@literal <caio.luke at artelys.com>}
  */
 public interface DecomposedFlowRescaler {
-    DecomposedFlow rescale(DecomposedFlow decomposedFlow);
+    DecomposedFlow rescale(DecomposedFlow decomposedFlow, Network network);
```

Comment:
We have a last suggestion:
```suggestion
    default DecomposedFlow rescale(DecomposedFlow flow) {
        return rescale(flow, null);
    }

    DecomposedFlow rescale(DecomposedFlow decomposedFlow, Network network);
```
This will avoid to add a breaking change.
You can also revert the change in the implementation that does not need a `network`.
This will also avoid using `null` in some tests.

Reply:
Well spotted :)

> You can also revert the change in the implementation that does not need a network.
I'm not sure I understand, do I still need to change something else?

Reply:
You did well ! I mean to revert changes in tests (not the implementation)